gtest/src/system.rs

```diff
@@ -157,6 +172,7 @@ impl System {
         self.0.borrow_mut().store_new_code(&code)
     }
 
+    /// Get mailbox by program id.
```

Comment:
Make these comments more alive pls

for example,
"Get mailbox by program id" -> "Extract mailbox of user with given id"

Reply:
"If `id` is an active program, returns `true`" -> "Returns bool defining if program with given id is active. Active program means that it could be called etc....."